packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -587,33 +588,33 @@ void _applyCameraUpdate(gmaps.GMap map, CameraUpdate update) {
     case 'newCameraPosition':
       final Map<String, Object?> position = asJsonObject(json[1]);
       final List<Object?> latLng = asJsonList(position['target']);
-      map.heading = position['bearing'] as num?;
-      map.zoom = position['zoom'] as num?;
+      map.heading = position['bearing']! as num;
+      map.zoom = position['zoom']! as num;
       map.panTo(
-        gmaps.LatLng(latLng[0] as num?, latLng[1] as num?),
+        gmaps.LatLng(latLng[0]! as num, latLng[1]! as num),
       );
-      map.tilt = position['tilt'] as num?;
+      map.tilt = position['tilt']! as num;
     case 'newLatLng':
       final List<Object?> latLng = asJsonList(json[1]);
-      map.panTo(gmaps.LatLng(latLng[0] as num?, latLng[1] as num?));
+      map.panTo(gmaps.LatLng(latLng[0]! as num, latLng[1]! as num));
     case 'newLatLngZoom':
       final List<Object?> latLng = asJsonList(json[1]);
-      map.zoom = json[2] as num?;
-      map.panTo(gmaps.LatLng(latLng[0] as num?, latLng[1] as num?));
+      map.zoom = json[2]! as num;
+      map.panTo(gmaps.LatLng(latLng[0]! as num, latLng[1]! as num));
     case 'newLatLngBounds':
       final List<Object?> latLngPair = asJsonList(json[1]);
       final List<Object?> latLng1 = asJsonList(latLngPair[0]);
       final List<Object?> latLng2 = asJsonList(latLngPair[1]);
       final double padding = json[2] as double;
       map.fitBounds(
         gmaps.LatLngBounds(
-          gmaps.LatLng(latLng1[0] as num?, latLng1[1] as num?),
-          gmaps.LatLng(latLng2[0] as num?, latLng2[1] as num?),
+          gmaps.LatLng(latLng1[0]! as num, latLng1[1]! as num),
+          gmaps.LatLng(latLng2[0]! as num, latLng2[1]! as num),
```

Comment:
These changes look risky, going from something that may be nullable `as num?` to asserting without checking that they're not-null `[index]!`.